packages/auth/src/auth-options.ts

```diff
@@ -1,9 +1,9 @@
 import { prisma } from "@agreeto/db";
 import { PrismaAdapter } from "@next-auth/prisma-adapter";
 import type { NextAuthOptions } from "next-auth";
-import GoogleProvider from "next-auth/providers/google";
-import AzureAdProvider from "next-auth/providers/azure-ad";
 import { refreshAccessToken } from "./refresh-access-token";
+import { getGoogleProvider } from "./providers/google";
+import { getAzureAdProvider } from "./providers/azure";
```

Comment:
The options object started to get really large when I added the scope so I thought I'd move them out.

We could skip the idea of a bunch of getters though, I think it's fine to use process.env since they will still be validated (from the next.js app)